twisted/protocols/policies.py

```diff
@@ -651,7 +651,8 @@ def __init__(self, wrappedFactory, logfilePrefix, lengthLimit=None):
 
 
     def open(self, name):
-        return file(name, 'w')
+        import builtins
+        return builtins.open(name, 'w')
```

Comment:
Is this dance needed? I don't think that `open` is aliased here, so it should be safe simply to have `return open(name, 'w')`. Of course, it's hard to know, because this code appears to be uncovered.